mycore-base/src/main/java/org/mycore/datamodel/metadata/MCRDerivate.java

```diff
@@ -50,14 +50,16 @@ public final class MCRDerivate extends MCRBase {
 
     public static final String ROOT_NAME = "mycorederivate";
 
+    public static final String OBJECT_TYPE = "derivate";
```

Comment:
Perhaps it would be better to decouple the object type into a separate class: `MCRObjectTypes`?

Reply:
I prefer it at class level. The "derivate" object type is special. In a usual application you have individual object types. So having a MCRObjectTypes class doesn't really makes sense cause you don't know any of the application specific ones.